pkgs/tools/misc/bashcards/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchFromGitHub
+, stdenv
```

Comment:
Swap these two (put stdenv on the first line).

Reply:
Will do!